### DISCUSSION TITLE
Hover card unit testing

### DISCUSSION THREAD

**Discussion by huylevuanh** [radix-ui/primitives#2013] [Help]
## Question
How to write unit test for hover card component with Vitest and  React-testing-library, I can not stimulate hover event

**Comment by joaom00** [comment]
You should mock `ResizeObserver` and `HoverCard` have a default delay of 700ms to open, so you need to use `waitFor`

An example https://github.com/joaom00/radix-select-vitest/blob/main/src/HoverCard.test.tsx

**Comment by huylevuanh** [comment]
We smashed it, thank a lot
